onadata/apps/viewer/migrations/0003_auto_20171123_1521.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('viewer', '0002_auto_20160205_1915'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='export',
+            name='export_type',
+            field=models.CharField(default=b'xls', max_length=10, choices=[(b'xls', b'Excel'), (b'csv', b'CSV'), (b'gdoc', b'GDOC'), (b'zip', b'ZIP'), (b'kml', b'kml'), (b'csv_zip', b'CSV ZIP'), (b'sav_zip', b'SAV ZIP'), (b'sav', b'SAV'), (b'external', b'Excel'), (b'analyser', b'Analyser')]),
```

Comment:
It looks like the only change here from https://github.com/kobotoolbox/kobocat/blob/44a797c5010485cc54e53e8a1acd41c26934e312/onadata/apps/viewer/migrations/0001_initial.py#L31 is adding `(b'analyser', b'Analyser')` as a choice. This isn't enforced at a database level, so let's omit this auto-generated migration altogether. Alternatively, we could leave the migration but give it a more descriptive name.

Reply:
I decided to add it to the PR because it was automatically added with all databases I've tested (even production ones).  
Because I saw several other "auto" migrations, I though I had to keep that name to distinguish  auto migrations from others. 
I would keep it to be consistent with the code and give it a better name.

Reply:
I'm just going to merge it as-is.